File: while.p.go
package main

import (
	"os"
	"strconv"
	"time"
)

func isPrime(n int) bool {
	i := 3
	for i < n/2 {
		if n%i == 0 {
			return false
		}

		i += 2
	}

	return true
}

func main() {
	max, _ := strconv.Atoi(os.Args[len(os.Args)-1])

	start := time.Now()

	prime := 0
	i := 1
	for i < max {
		if isPrime(i) {
			prime++
		}

		i += 2
	}

	println(time.Since(start).Milliseconds())

	_ = prime
}

File: while.p.js
const isPrime = (n) => {
	let i = 3;
	while (i < n / 2) {
		if (n % i === 0) {
			return false;
		}

		i += 2;
	}

	return true;
};

const max = +process.argv[process.argv.length - 1];

const start = Date.now();
let prime = 0;

let i = 1;
while (i < max) {
	if (isPrime(i)) {
		prime++;
	}

	i += 2;
}

console.log(Date.now() - start);
